wavefront_cli/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.115'
+__version__ = '0.0.116'
```

Comment:
It might be worth to add a Travis CI check for the version update inside every PR
```
  - if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
      test -n "$(git diff -G __version__ $TRAVIS_COMMIT_RANGE wavefront_cli/__init__.py)";
    fi
```
https://github.com/wavefrontHQ/wavefront-sdk-python/blob/master/.travis.yml#L22